nhrpd/nhrp_route.c

```diff
@@ -456,10 +438,11 @@ void nhrp_send_zebra_nbr(union sockunion *in,
 		return;
 	s = zclient->obuf;
 	stream_reset(s);
-	zclient_neigh_ip_encode(s, out ? ZEBRA_NEIGH_IP_ADD :
-				ZEBRA_NEIGH_IP_DEL, in, out,
-				ifp, out ? ZEBRA_NEIGH_STATE_REACHABLE
-				: ZEBRA_NEIGH_STATE_FAILED);
+	zclient_neigh_ip_encode(s, out ? ZEBRA_NEIGH_IP_ADD : ZEBRA_NEIGH_IP_DEL,
+				in, out, ifp,
+				out ? ZEBRA_NEIGH_STATE_REACHABLE
+				    : ZEBRA_NEIGH_STATE_FAILED,
+				0);
```

Comment:
please add comment of the meaning of 0 (hardcoded value)
something like 

0); // meaning something cool